docs/monorepo.md

```diff
@@ -1,77 +1,323 @@
-**Overview**
-# The Catalyst monorepo
+**Developers**
 
-## Primary contents
+# Getting started with Catalyst
+
+How you get started with Catalyst depends on your specific goals. You might be:
+
+1. A **Storefront Developer**, looking to create a custom storefront using Catalyst as your foundation.
+2. An **Integration Developer**, aiming to build add-ons or integrations that can be optionally included in Catalyst storefronts.
+3. A **Contributor**, interested in enhancing the Catalyst project by submitting pull requests for new features, bug fixes, or other improvements.
+
+Regardless of your role, the [main GitHub repository for Catalyst](https://github.com/bigcommerce/catalyst) is the ideal starting point.
+
+The initial setup steps are the same for all developers, and this guide will later provide tailored instructions based on your specific path.
+
+## Overview
 
 The [main GitHub repository for Catalyst](https://github.com/bigcommerce/catalyst) is a [monorepo](https://en.wikipedia.org/wiki/Monorepo) with multiple codebases, including but not limited to the following:
 
-* The [core Next.js reference storefront application, in /core](https://github.com/bigcommerce/catalyst/tree/main/core)
-* [The component library, in /core/components/ui](https://github.com/bigcommerce/catalyst/tree/main/core/components/ui)
-* [The API client, in /packages/client](https://github.com/bigcommerce/catalyst/tree/main/packages/client)
-* [Product documentation, in /docs](https://github.com/bigcommerce/catalyst/tree/main/docs)
+- [The core Next.js reference storefront application, in `/core`](https://github.com/bigcommerce/catalyst/tree/main/core)
+- [Packages published to NPM, in `/packages`](https://github.com/bigcommerce/catalyst/tree/main/packages/client)
+- [Product documentation, in `/docs`](https://github.com/bigcommerce/catalyst/tree/main/docs)
 
-### The monorepo and the CLI
+It's important to note that we use several tools throughout the Catalyst monorepo, such as [Turborepo](https://turbo.build/repo/docs) and [pnpm Workspaces](https://pnpm.io/workspaces); you'll notice references to these tools as you spend more time in the repository, but most developers likely won't need to change any files, code, or configuration related to these tools in their normal Catalyst workflows.
 
-The [Catalyst CLI](https://www.npmjs.com/package/@bigcommerce/create-catalyst) installs just the Next.js application from `core/` onto your computer when you create a new Catalyst storefront.
+## Prerequisites
 
-If you are interested solely in building a headless storefront based on Catalyst, most of this monorepo is not relevant to you.
+1. **Node.js version `20.x`:** Catalyst currently supports Node.js version `20.x`, and does not support versions `<= 19` or `>= 21` ([click here to download Node.js](https://nodejs.org/en/download/package-manager))
```

Comment:
I don't think we should refer to odd node versions as they are not stable and can cause setup issues (that we've seen). I see that you mention we *don't* support them, but might just be best to omit it completely.